modules/evm-accounts/src/lib.rs

```diff
@@ -260,7 +278,7 @@ where
 	T::AccountId: From<AccountId32>,
 {
 	fn into_h160(account_id: AccountId32) -> H160 {
-		EvmAddresses::<T>::get(&Into::<T::AccountId>::into(account_id))
+		EvmAddresses::<T>::get(&Into::<T::AccountId>::into(account_id)).unwrap_or_default()
```

Comment:
@xlc is it safe to return empty H160?

Reply:
This was the old behavior

https://github.com/AcalaNetwork/Acala/issues/525 to cleanup this